### ISSUE TITLE
DbFunctions.CreateDateTime yields NotSupportedException

### ISSUE THREAD

**Issue by meoblast001** [npgsql/EntityFramework6.Npgsql#118]
Reproduction steps: Create any context with an active database connection. Assuming the context contains a DbSet "Entities", try the following:

    Console.WriteLine(ctx.Entities.Select(x => DbFunctions.CreateDateTime(1, 1, 1, 1, 1, 1)).FirstOrDefault());

An exception is thrown: "NotSupportedException: NotSupported CreateDateTime"

**Comment by lfsaestrella** [comment]
Hello! I had the same issue as you. Did you manage to solve it?

**Comment by meoblast001** [comment]
I don't believe there is a workaround. The driver doesn't implement that DbFunction.
